### ISSUE TITLE
[Enhancement] Character count

### ISSUE THREAD

**Issue by janxkoci** [lainsce/quilter#169]
The word count in status bar is very useful, however sometimes I also need to know the number of characters in a document. Copying text to other app (LO, gedit) to get this info completely breaks the distraction-free workflow this app aims to create.

This could be implemented either as a separate item on the status bar (there is enough room) or as a popover from the word counter with extra info. It should also update count if I select only part of text (this is also missing in the current word counter). 

So to sum up, this info should be available:
- Count of all characters (including spaces)
- Count of all characters (without spaces, less important*)
- Update counter for text selections (also word counter should do this)

**I never used this info, but it's usually present in packages like LO or gedit, so I assume somebody may need it too.*

Thanks for considering this enhancement, I believe it would make this great app even more useful to a lot of people.

**Comment by lainsce** [comment]
I'll see what I can do :smile:

**Comment by cicindel** [comment]
Yes, better than nothing ;)

**Comment by lainsce** [comment]
Then I'm on it~

**Comment by lainsce** [comment]
![quilter_track_choice_popup](https://user-images.githubusercontent.com/4886639/45600338-7fe7c800-b9d1-11e8-8c39-de498aecdcf4.png)

I made a mockup of what it'll look like.
